backend/main/server/controllers.go

```diff
@@ -135,7 +135,7 @@ func (a *App) createVoteForProposal(w http.ResponseWriter, r *http.Request) {
 
 	vote, err := helpers.createVote(r, proposal)
 	if err != nil {
-		respondWithError(w, http.StatusInternalServerError, err.Error())
+		respondWithError(w, http.StatusForbidden, err.Error())
```

Comment:
if `helpers.createVote` hits the lines below then we will return forbidden when really its just an invalid payload

https://github.com/DapperCollectives/CAST/blob/ceddc9a86bf85bddfe934dcd0687379c7e39af03/backend/main/server/helpers.go#L288-L292